src/components/biggive-social-icon/biggive-social-icon.tsx

```diff
@@ -23,6 +23,11 @@ export class BiggiveSocialIcon {
    */
   @Prop() iconColour: string = 'white';
 
+  /**
+   * Wide?
+   */
+  @Prop() wide: boolean = false;
```

Comment:
this docbloc doesn't help me much - it's just repeating the name of the prop. Can we either explain what wide means or not have a docblock at all? I think some explanation might be useful - why are icons sometimes wide and sometimes not wide?

Reply:
Good shout, fixed.